async-csharp/AwaitInForEachLoops/Tests/TaskWhenAllTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AwaitInForEachLoop;
```

Comment:
Unnecessary using, this is already in the global usings

Reply:
removed unused using